### PR TITLE
[stable/jenkins]  Added option to disable security on master node

### PR DESCRIPTION
**Pull request by electroma**
Added option to turn off security on master.
May be a good option for 
-smaller teams running private cluster.
-test installations

### CODE REVIEW

File: stable/jenkins/templates/jenkins-master-deployment.yaml [hunk 1]
```diff
@@ -53,6 +53,7 @@ spec:
           env:
             - name: JAVA_OPTS
               value: "{{ default "" .Values.Master.JavaOpts}}"
+            {{- if .Values.Master.UseSecurity }}
```
Comment (bacongobbler) [inline stable/jenkins/templates/jenkins-master-deployment.yaml:56]: we'd probably want to wrap the secret as well such that `jenkins-admin-password` and `jenkins-admin-username` can be omitted (or the secret omitted entirely).

Comment (electroma) [reply stable/jenkins/templates/jenkins-master-deployment.yaml:56]: @bacongobbler fixed and tested

**Comment by electroma** [comment]
@bacongobbler please review / approve

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

**Pull request merged**